### ISSUE TITLE
Using with gulp-sourcemaps

### ISSUE THREAD

**Issue by subinvarghesein** [Josiah/gulp-cachebust#9]
Following is my relevant code:

    gulp.task('scripts', ['clean'], function() {
        return gulp.
            src('src/**/*.js').
            pipe(sourcemaps.init()).
                pipe(uglify()).
                pipe(concat('script.js')).
            pipe(sourcemaps.write('.')).
            pipe(cachebust.resources()).
            pipe(gulp.dest("/dist"));
    });

Now, the files generated are: `script.12345678.js` and `script.js.87654321.map`

In short, cachebust token is getting added to the sourcemap file but inside `script.js`, sourcemap url is `script.js.map`. 

So how should this be handled?

**Comment by Deadarius** [comment]
Please try to put ```pipe(cachebust.resources()).``` before ```pipe(sourcemaps.write('.')).```

**Comment by joekaiser** [comment]
I am having the same issue. I make sure to write out the sourcemap before I call .resources(), but the sourcemap filename is still changed.

```
gulp.task('build-js', ['clean-dist'], function (cb) {
    pump([
        gulp.src(['./public/app.js', './public/**/*.js']),
        sourcemaps.init(),
        concat('site-scripts.js'),
        uglify(),
        sourcemaps.write('./'),
        cachebust.resources(),
        gulp.dest('./public/dist')
    ],
        cb
    );
});

```
